#status of the robot

#robot id
int32 id

#robot name
string name

#robot state
int32 state

#pose of the robot
geometry_msgs/Pose pose

#velocity of the bot
geometry_msgs/Twist twist

#storage used
int32 storage_used

#storage capacity
int32 storage_capacity

#robot type
bool type
